_posts/2019-05-16-released-v6.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Released Chainer/CuPy v6.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v6.0.0 today!
+This is a major release that introduces several new features.
+Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tag/v6.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v6.0.0).
+
+## ChainerX
+
+The biggest update is the introduction of **ChainerX**.
+It is a fast and portable ndarray engine with autograd support written in C++ with very thin Python wrapper.
+
+We have released the beta version of ChainerX in v6.0.0b1 as we wrote in the [previous blog post](https://chainer.org/announcement/2018/12/03/chainerx.html).
+Since then, we have been working on improving it in various aspects.
+In particular, ChainerX in v6.0.0 expands the coverage of various features since v6.0.0b1.
+
+- **More op coverage**.
```

Comment:
I think this is a more common way to put it. Ditto for the following similar occurrences.
```suggestion
- **Wider op coverage**.
```